Assets/Scripts/Player/HouseInputManager.cs

```diff
@@ -221,7 +229,10 @@ private void RotateDecorateCamera(Vector2 mouseDelta)
 		HouseManager.Instance.DecorateCamera.transform.localRotation = Quaternion.Euler(HouseManager.Instance.DecorateCamera.transform.localRotation.eulerAngles.x, cameraYRotation, 0);
 	}
 
-	// TODO - get explanation from Kai?
+	// REVIEW - get explanation from Kai?
```

Comment:
I'm unsure of the need for the near and far camera `Vector3`s. This seems like it could be simplified by just using `camera.ScreenPointToRay(Mouse.current.position.ReadValue())` for the ray and then raycasting as usual with the layer mask.